<launch>
  
  <node name="predator_node" type="predator_node" pkg="pandora_vision_predator" 
      respawn="False" output="screen" />
  <!-- Add in upper line to enable debugging: launch-prefix="kdbg" /-->
  
  <rosparam file="$(find pandora_vision_predator)/config/predator_topics.yaml" 
        command="load" ns="predator"/>
        
  <rosparam file="$(find pandora_vision_predator)/config/predator_params.yaml" 
        command="load" ns="predator"/>
        
  <param name="pattern_path" type="str" value="$(find pandora_vision_predator)/model"/>
  <param name="export_path" type="str" value="$(find pandora_vision_predator)/model"/>

</launch>
